### ISSUE TITLE
error selecting column of vertica datatype long varchar

### ISSUE THREAD

**Issue by bicyclethief** [wvanbergen/vertica#34]
When selecting a column of data type *long var* from Vertica 7, I'm seeing the following error:

```
../vertica-0.11.4/lib/vertica/column.rb:54:in `initialize': undefined method `[]' for nil:NilClass (NoMethodError)
```

I believe this code can reproduce the issue:

```ruby
require 'vertica'

connection = Vertica.connect({
  :host     => 'db_server',
  :user     => 'user',
  :password => 'password',
  # :ssl         => false, # use SSL for the connection
  # :port        => 5433,  # default Vertica port: 5433
  # :database    => 'db',  # there is only one database
  # :role        => nil,   # the (additional) role(s) to enable for the user.
  # :search_path => nil,   # default: <user>,public,v_catalog
  # :row_style   => :hash  # can also be :array (see below)
})

result = connection.query('DROP TABLE IF EXISTS public.long_varchar_test')
p result

result = connection.query('CREATE TABLE IF NOT EXISTS public.long_varchar_test(id  IDENTITY,good_column int NOT NULL,bad_column long varchar);')
p result

result = connection.query('SELECT good_column from public.long_varchar_test')
p result

result = connection.query('SELECT bad_column from public.long_varchar_test')
p result

connection.close
```

**Comment by wvanbergen** [comment]
@bicyclethief Can you give a simple `CREATE TABLE` example that will trigger the problem?

**Comment by wvanbergen** [comment]
We're probably just missing an entry in this table: https://github.com/wvanbergen/vertica/blob/master/lib/vertica/column.rb#L24-L44

In #41, I added some better error handling code, so we know what the OID of the type is.

**Comment by wvanbergen** [comment]
Fixed by #34.

**Comment by bicyclethief** [comment]
@wvanbergen Thanks!
